_posts/2020-12-02-summer-2020-newsletter.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Newsletter, summer 2020"
+date: 2020-12-02
+author: Elena Findley-de Regt
+type: blogpost
+excerpt: Our first newsletter, bumper summer edition!
+categories:
+  - News
+---
+# Foundation for Public Code newsletter - summer 2020 
+
+> This newsletter was originally emailed to subscribers on 31 August 2020. [Subscribe to our newsletter](https://forms.gle/gn7wR2Eaxbv5g1BF9) to get the next one via email!
+
+Hello,
+
+Welcome to our newsletter, bumper summer edition. Like last summer, many of the people we work with went on holiday, but that hasn't slowed us down.
+
+🏅 Perfectly timed to get us pumped for summer, we got an Encouragement Award from the Netherlands Internet Society (as part of their annual Innovation Award) in June. [Read more on our blog](https://blog.publiccode.net/news/2020/06/17/isoc-encouragement-award-consider-us-encouraged.html).
+
+## What we've been up to 👩‍💻 🧑🏻‍🤝‍🧑🏽 🤙
+
+### Codebase stewardship
+
+* We currently have 2 codebases under stewardship: [Signalen](https://publiccode.net/codebases/signalen.html) and [OpenZaak](https://publiccode.net/codebases/openzaak.html). Both codebases are focused on growing their open source communities in 2020.
+* In June, we built a prototype product marketing website fo OpenZaak: <openzaak.org>.
```

Comment:
Should this be a link to openzaak.org?

Reply:
Yup! In other markdown, this generates a link.